lib/index.js

```diff
@@ -0,0 +1,2 @@
+require("babel/register");
```

Comment:
This might be better at the root of the project since it is not really a lib file